modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/sql/GridSqlQuerySplitter.java

```diff
@@ -1860,6 +1860,9 @@ private void splitAggregate(
 
                 break;
 
+            case UNKNOWN_FUNCTION:
```

Comment:
Why UNKNOWN_FUNCTION is used as a type for user aggregate functions?

Reply:
UNKNOWN_FUNCTION is used for custom functions.
The idea is to use UNKNOWN_FUNCTION for all custom functions including aggregate.
@tledkov-gridgain, Does it make sense?

Reply:
Can this break non-aggregate functions?
Do we have a test for non-collocated case using non-aggregate custom functions?

Reply:
It can't break non-aggregate functions since they don't participate in map/reduce scenario.

Reply:
There is test IgniteCacheAbstractQuerySelfTest#testUserDefinedFunction for custom functions